addons/bestja_frontend_fpbz/static/src/css/custom.css

```diff
@@ -0,0 +1,569 @@
+    src: url('fonts/OpenSans-CondBold-webfont.eot?#iefix') format('embedded-opentype'),
+        url('fonts/OpenSans-CondBold-webfont.woff') format('woff'),
+        url('fonts/OpenSans-CondBold-webfont.ttf') format('truetype'),
+        url('fonts/OpenSans-CondBold-webfont.svg#open_sans_condensedbold') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'open_sanscondensed_light';
+    src: url('fonts/OpenSans-CondLight-webfont.eot');
+    src: url('fonts/OpenSans-CondLight-webfont.eot?#iefix') format('embedded-opentype'),
+        url('fonts/OpenSans-CondLight-webfont.woff') format('woff'),
+        url('fonts/OpenSans-CondLight-webfont.ttf') format('truetype'),
+        url('fonts/OpenSans-CondLight-webfont.svg#open_sanscondensed_light') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'open_sansCnLtIt';
+    src: url('fonts/OpenSans-CondLightItalic-webfont.eot');
+    src: url('fonts/OpenSans-CondLightItalic-webfont.eot?#iefix') format('embedded-opentype'),
+        url('fonts/OpenSans-CondLightItalic-webfont.woff') format('woff'),
+        url('fonts/OpenSans-CondLightItalic-webfont.ttf') format('truetype'),
+        url('fonts/OpenSans-CondLightItalic-webfont.svg#open_sansCnLtIt') format('svg');
+    font-weight: normal;
+    font-style: normal;
+}
+/* End of Open Sans Fonts */
+
+/* Common css */
+body{
+    font-family: "open_sanscondensed_light";
+}
+
+.open_sans_condensedbold{
+    font-family: "open_sans_condensedbold";
+    font-weight: 700;
+}
+
+.color_orange{
+    color: #F29000;
+}
+
+/* Centered columns styles */
+.row-centered {
+    text-align: center;
+}
+.col-centered {
+    display: inline-block;
+    float: none;
+    /* reset the text-align */
+    text-align: left;
+    /* inline-block space fix */
+    margin-right: -4px;
+}
+/* End of Centered columns styles */
+/* End of Common css */
+
+/* FPBZ Theme module description */
+.theme_description{
+    overflow: hidden;
+}
+/* End of FPBZ Theme module description */
+
+/* Main top snippet */
+.main_top_white_text{
+    color: white;
+    font-style: normal;
+    margin-top: 95px;
+}
+
+.main_top_white_text h1{
+    font-size: 36pt;
+}
+
+.differentiator_1{
+    font-size: 46pt;
+}
+
+@media screen and (max-width: 991px) {
+    .main_top_white_text h1, .choose_place{
+        font-size: 26pt;
+    }
+
+    .differentiator_1{
+        font-size: 36pt;
+    }
+}
+
+@media screen and (max-width: 767px) {
+    .main_top_white_text h1, .choose_place{
+        font-size: 20pt;
+    }
+
+    .differentiator_1{
+        font-size: 26pt;
+    }
+}
+
+@media screen and (max-width: 480px) {
+    .main_top_white_text h1, .choose_place{
+        font-size: 12pt;
+    }
+
+    .differentiator_1{
+        font-size: 16pt;
+    }
+}
+
+.main_top_photo_text{
+    /*    background: url('/bestja_frontend_fpbz/static/src/img/content/pan_z_talerzem.jpg') 
```

Comment:
This commented code should be removed – file `pan_z_talerzem.jpg` is not present in the repository anyway.